### PR TITLE
Next

### PR DESCRIPTION
**Pull request by Hosch250**

### CODE REVIEW

File: RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        {
+            get { return _target; }
+            set
+            {
+                _target = value;
+                if (_target == null)
+                {
+                    return;
+                }
+            }
```
Comment (ckuhn203) [inline RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.cs]: I don't quite understand what you're doing here.  You're returning at the same time the method would return anyway. Did you mean to check the `value` for null prior to setting?? If so, an argument exception should be thrown, no?

File: RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx [hunk 2]
```diff
@@ -0,0 +1,142 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ckuhn203) [inline RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx]: Do we really want to mix and match XAML with Winforms? If we decide this dialog should be dockable later, this entire UI would need rebuilt and ported to Winforms. That may be unlikely, but something to consider.

File: RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersPresenter.cs [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+
+                // This is to prevent throws when this statement fails:
+                // (VBAParser.ArgsCallContext)proc.argsCall();
+                try
+                {
+                    var check = (VBAParser.ArgsCallContext)proc.argsCall();
+                }
+                catch
+                {
+                    // update letter/setter methods - needs proper fixing
```
Comment (ckuhn203) [inline RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersPresenter.cs]: Needs a proper `TODO:` comment

File: RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx (hunk 2 above)
Comment (retailcoder) [inline RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx]: @ckuhn203  resx is the WinForms format for resource files.... FYI WPF localization ditches the .resx format :wink:

Comment (ckuhn203) [inline RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx]: D'oh! :+1:

File: RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.cs (hunk 1 above)
Comment (retailcoder) [inline RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.cs]: R# marked this as dead code, it's gone.

**Pull request closed without merging**